crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -695,6 +695,11 @@ pub struct ExecutionArgs {
     #[clap(short = 'F', long, group = "scope-filter-group")]
     pub filter: Vec<String>,
 
+    /// Run only tasks that are affected by changes between
+    /// the current branch and `main`
```

Comment:
Is `main` configurable?

Reply:
yes, with the `TURBO_SCM_BASE` env var

Reply:
Could it be an optional argument to `--affected=<base>` also? Not a huge deal, but it would be similar to `--filter` (with much lower surface area)